### ISSUE TITLE
If WPR was installed before running test, it won't be deleted when run the tests

### ISSUE THREAD

**Issue by Mai-Saad** [wp-media/wp-rocket-e2e#153]
steps:
1- install and activate WPR on e2e env manually
2- clone e2e 
3- on dev branch
4- npm i 
5- npm run test:smoke 

this screenshot is not handled in e2e , we need to handle that if it's there
![Image](https://github.com/user-attachments/assets/441ce873-ff36-4a2b-b2f1-7b1c0fe86302)
or 
![Image](https://github.com/user-attachments/assets/c3e64fed-0514-40c6-9db4-76de2dc6edee)

**Comment by Mai-Saad** [comment]
Acceptance criteria:
- Smoke test is passing
- lcp test is passing
- lrc is passing (some templates may have VR  issue, this is out of scope here)

**Comment by MathieuLamiot** [comment]
https://wp-media.slack.com/archives/C05NH7JU4S2/p1729169954462949

**Comment by MathieuLamiot** [comment]
Blocked by: https://github.com/wp-media/wp-rocket-e2e/issues/163
To re-check after it.

**Comment by MathieuLamiot** [comment]
Looks fixed.
